lib/db.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var sqlGen = require('./sql_gen.js');
+var configValidator = require('./config_validator.js');
+
+var methods = { init: function (client, config, _, cb) {
+  configValidator(config);
+
+  return client.query(sqlGen.init(config), cb);
+} };
```

Comment:
does this pass the linter?
(_thought it would require the `init` to be on a new line and these closing brackets too... if not. fair enough..._)